lms/djangoapps/teams/search_indexes.py

```diff
@@ -88,7 +88,7 @@ def search_is_enabled(cls):
         return settings.FEATURES.get(cls.ENABLE_SEARCH_KEY, False)
 
 
-@receiver(post_save, sender=CourseTeam)
+@receiver(post_save, sender=CourseTeam, dispatch_uid='teams.signals.course_team_post_save_callback')
```

Comment:
There was something strange happening when registering the signal without it.  It would successfully disconnect and reconnect the for the first test it ran, then for all subsequent tests it would not disconnect the signal. (It also was in a different place in the receiver stack, not sure if that makes a difference).  Explicitly giving the signal a `dispatch_uid` made the connecting and disconnecting work every time.